lang/en/docs/_installations/mac.md

```diff
@@ -9,6 +9,12 @@ This will also install Node.js if it is not already installed.
 brew install yarn
 ```
 
+If you use [nvm](https://github.com/creationix/nvm) you should exclude installing NodeJS so that the nvm binaries are used.
```

Comment:
Maybe change "nvm binaries" to "nvm's version of Node.js"?

Also, small nit: Use "Node.js", not "NodeJS".